<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmDefence</class>
 <widget class="QDialog" name="frmDefence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>防区管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/image/main.png</normaloff>:/image/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;宋体&quot;;</string>
  </property>
  <widget class="QGroupBox" name="gboxBottom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>1021</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QComboBox" name="txtSubSystem">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labSubSystem">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>子系统:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>防区号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtDefenceID">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="txtDefenceType">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>防区名称:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDefenceName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceType">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>防区类型:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>主机编号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtHostID">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="txtHostName">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labHostName">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>主机名称:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceUser">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>用户:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtDefenceUser">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>admin</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtDefenceY">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceX">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>10</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceY">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDefenceX">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>10</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceIndex">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>索引:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtDefenceIndex">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="txtDefenceImage">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>10</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceImage">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>地图:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="txtDefenceSound">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>40</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labDefenceSound">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>声音:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gboxMain">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="gboxRight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>修改(&amp;U)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/excel.png</normaloff>:/image/excel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>删除(&amp;D)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>添加(&amp;A)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMap">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>45</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>导入(&amp;O)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/input.png</normaloff>:/image/input.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>调整防区位置(&amp;M)</string>
    </property>
    <property name="icon">
     <iconset resource="main.qrc">
      <normaloff>:/image/move.png</normaloff>:/image/move.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QTreeWidget" name="treeDefence">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>471</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
 </widget>
 <tabstops>
  <tabstop>treeDefence</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnMap</tabstop>
  <tabstop>btnMove</tabstop>
  <tabstop>txtDefenceID</tabstop>
  <tabstop>txtSubSystem</tabstop>
  <tabstop>txtDefenceName</tabstop>
  <tabstop>txtDefenceType</tabstop>
  <tabstop>txtHostID</tabstop>
  <tabstop>txtHostName</tabstop>
  <tabstop>txtDefenceIndex</tabstop>
  <tabstop>txtDefenceSound</tabstop>
  <tabstop>txtDefenceImage</tabstop>
  <tabstop>txtDefenceUser</tabstop>
  <tabstop>txtDefenceX</tabstop>
  <tabstop>txtDefenceY</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections/>
</ui>
